### ISSUE TITLE
Can't use service with annotation.traefik.ingress.kubernetes.io/affinity: "true" after update traefik to v1.7.23

### ISSUE THREAD

**Issue by ankworld** [containous/traefik#6574]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

- Create service with annotation.traefik.ingress.kubernetes.io/affinity: "true"

### What did you expect to see?

- Traefik can run normally which v1.7.21 can run.

### What did you see instead?

Logging

- executing "\\" at <$backend.Stickiness.Secure>: can't evaluate field Stickiness in type *types.Backend"

Problem
- Rules hadn't been load.

### Output of `traefik version`: 1.7.23

```
Version:      v1.7.23
Codename:     maroilles
Go version:   go1.14.1
Built:        2020-03-23_04:04:41PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
[entryPoints]
  [entryPoints.http]
  address = ":80"
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
....
executing "\" at <$backend.Stickiness.Secure>: can't evaluate field Stickiness in type *types.Backend"
....
```

**Comment by ankworld** [comment]
my example.yaml

```yaml
kind: Service
apiVersion: v1
metadata:
  labels:
    app: auth
  name: auth
  annotations:
    traefik.ingress.kubernetes.io/affinity: "true"
spec:
  type: ClusterIP
  ports:
  - port: 80
    targetPort: 80
  selector:
    app: auth
```

**Comment by traefiker** [comment]
Closed by #6576.
